src/Http/Controllers/Controller.php

```diff
@@ -27,4 +28,18 @@ protected function createPaginatedResponse(
 
         return new JsonResponse($paginator);
     }
+
+    /**
+     * Get the user model based on the user id parameter.
+     *
+     * @return \App\Models\User
+     */
+    protected function getUserFromRoute(): Model
+    {
+        $userId = request()->route()->parameter('user');
+
+        $modelClass = config('gossip.user_model');
+
+        return $modelClass::findOrFail($userId);
+    }
```

Comment:
Might be nicer to use route resolver. What is the reason you went with this solution?

Reply:
How would that work if the user model is stored in a configuration value and we don't want to interfere the application which this package is in installed? I couldn't find something which I thought was useful in the docs, can you point me into a direction?

Reply:
Take a look at this solution:
https://github.com/owowagency/laravel-resources/blob/master/src/Controllers/ResourceController.php#L261